src/zimscraperlib/misc.py

```diff
@@ -1,8 +1,8 @@
-""" Miscelaneous utils"""
+from __future__ import annotations
 
-from typing import Optional
+""" Miscelaneous utils"""
```

Comment:
DocString should be kept at the beginning of the file

Reply:
Done